### PR TITLE
docs: add new get started document

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

Part of #140 

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -1,144 +1,140 @@
...
+```bash
+> kubectl version
 ```
 
-- Check Matrixone cluster status
-
-```shell
-kubectl get po -n matrixone
-```
+The server version should >= 1.16.
```
Comment (DanielZhangQD) [inline docs/getting_started.md:20]: >=1.18 is required in L7

File: docs/getting_started.md [hunk 2]
```diff
@@ -1,144 +1,140 @@
...
+
+## Teardown MO cluster
+
+To teardown the cluster, delete the mo object in k8s:
+
+```bash
+> kubectl -n mo get matrixonecluster
+NAME   LOG   DN    TP    AP    UI    VERSION            AGE
+mo     3     2     2                 nightly-c371317c   13m
+> kubectl -n mo delete matrixonecluster mo
```
Comment (DanielZhangQD) [inline docs/getting_started.md:138]: And stop the port-forward process in L125?

**Review by DanielZhangQD — APPROVED** [review]
Rest LGTM

**Comment by aylei** [comment]
> Should we make mo cluster configuration example with minio like `mo-cluster-minio.yml`, aws may like `mo-cluster-aws.yml` ?

`examples/mo-cluster.yaml` has not yet used in document yet, we may switch to it after we have CI to ensure the example YAML always work

**Pull request merged**